riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
Dear all,

After our discussion, I run the following test:

I modified the clocksi_test by inserting clean_cluster and build_cluster between tests (a total of 15 tests in this module)
I created a version where I carefully pick the keys to not overlap across tests in the module and thus avoid using the clean and build calls.

This are the results:

More than 20 minutes for the one with the calls
Around 50s for the other one

How should I proceed?

Best,
Manuel

> On 14 Sep 2015, at 17:55, Christopher S. Meiklejohn <notifications@github.com> wrote:
> 
> In riak_test/opbcounter_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39417678>:
> 
> > @@ -33,7 +37,7 @@ confirm() ->
> >      decrement_test(Nodes),
> >      transfer_test(Nodes),
> >      conditional_write_test(Nodes),
> > -    rt:clean_cluster(Nodes),
> > +    %rt:clean_cluster(Nodes),
> I completely agree with you. My major point was that we need to look into each test before removing the call to determine if it will introduce any accidental non-determinism.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39417678>.
>